go-controller/pkg/ovn/multihoming_test.go

```diff
@@ -160,6 +160,43 @@ func (em *secondaryNetworkExpectationMachine) expectedLogicalSwitchesAndPorts()
 					}
 				case ovntypes.Layer2Topology:
 					switchName = ocInfo.bnc.GetNetworkScopedName(ovntypes.OVNLayer2Switch)
+					managementIP := managementPortIP(subnet)
+
+					if em.gatewayConfig != nil {
+						// there are multiple mgmt ports in the cluster, thus the ports must be scoped with the node name
```

Comment:
No, we have a management port on each node right ? Hence, their names must feature the node name.

Reply:
sure, but why do you assume in this function you have more than one node? wouldn't that be up to the test case?

Reply:
The production code templates the management port names with the node name since there's a management port per node. Hence they're unique.

The test case is actually using a single node. But I need to tell the test what OVN entities to expect; in this case, a single management port, templated with the node name.

I can delete the comment if that's the source of the confusion.

Reply:
yeah I think it is just the comments there that are confusing